ndcube/ndcube_sequence.py

```diff
@@ -239,6 +240,67 @@ def explode_along_axis(self, axis):
         # creating a new sequence with the result_cubes keeping the meta and common axis as axis
         return self._new_instance(result_cubes, common_axis=new_common_axis, meta=self.meta)
 
+    def stack_data(self):
+        """Generates a stacked array from a sequence of cubes."""
```

Comment:
more docs needed here of what the returned arrays are i.e. Returns : data, unites uncertainties